### ISSUE TITLE
Doxygen access issues over VPN

### ISSUE THREAD

**Issue by ndellingwood** [trilinos/Trilinos#11850]
## Bug Report
@trilinos/framework 

Access to Doxygen when on VPN is currently not working, for example trying to access trilinos.org/docs/dev/packages/tpetra/doc/html/index.html results in a short hang followed by error message "The connection was reset..." from Firefox ( no luck with Safari either)

**Comment by GrahamBenHarper** [comment]
@ndellingwood I think I see something similar on the SRN on-site. The link you posted is a Github page that doesn't exist, but on my end sites like https://trilinos.org/docs/dev/packages/tpetra/doc/html/index.html behave as you described.

**Comment by jhux2** [comment]
@ndellingwood I use the documentation from https://docs.trilinos.org/dev/packages/tpetra/doc/html/.  Does this work for you?

**Comment by ndellingwood** [comment]
@GrahamBenHarper @jhux2 yeah, I messed up the link I pasted but with the corrected links you each shared I have the same problem (over VPN). On my home computer I do not see the issue

**Comment by GrahamBenHarper** [comment]
@jhux2 that link also does not work on the SRN in CSRI

**Comment by ndellingwood** [comment]
I'm still having this issue today, I'm assuming it isn't some intermittent network problem?

**Comment by GrahamBenHarper** [comment]
@ndellingwood I've noticed this since Feb 15 or earlier, but I assumed it was a problem on my side with network/proxy configurations since other coworkers could access it when I mentioned it at the time.

**Comment by GrahamBenHarper** [comment]
@ndellingwood I just noticed this appears to be functional again. Can you verify if it works on your end as well?

**Comment by ndellingwood** [comment]
@GrahamBenHarper yeah, it is working for me now as well :)

@jhux2 thanks for the email with guidance for proxy settings, very helpful info (the doxygen links are now working for me without needing to do that)
